packages/docs/docs/cli/install.md

```diff
@@ -5,6 +5,10 @@ sidebar_label: install
 crumb: CLI Reference
 ---
 
+_removed in v4.0.0_
+
+_See [no more FFmpeg install](../4-0-migration#no-more-ffmpeg-install-ffmpegexecutable-option-removed)._ Here for archive purpose.
```

Comment:
```suggestion
_See: ["No more FFmpeg installation"](/docs/4-0-migration#no-more-ffmpeg-install-ffmpegexecutable-option-removed)._ Here for archival purpose.
```

Unfortunately Docusaurus has a bug where relative links don't work when you append `/` to the end of the documentation page URL which can happen automatically sometimes. So we have to absolutely specify it (without the domain)